localnet/poktrolld/config/app.toml

```diff
@@ -32,7 +32,7 @@ query-gas-limit = "0"
   enable = true
 
 [mempool]
-  max-txs = 5000
+  max-txs = -1
```

Comment:
Is this intentional?

According to [the docs](https://docs.cosmos.network/main/learn/advanced/config), this disables the mempool but I don't see why we would want to do that.

Reply:
@bryanchriswhite this is the value cosmos-sdk generates by default now. I, personally, like to keep the default values. And we do adjust this config on the live network.  

Related PRs explaining the change:

https://github.com/cosmos/cosmos-sdk/pull/20008 (original: https://github.com/cosmos/cosmos-sdk/pull/19970)
https://github.com/cosmos/cosmos-sdk/pull/20012

Reply:
@bryanchriswhite  To be clear, I do not oppose using the `5000` value here! :) Reading these two PRs still leaves the question of what the expectation for this value is on the full-blown network. I guess, just like with the existing mainnet, we will learn the behavior of the network and adjust.